resources/css/index.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Yeseva One";
+  src: url(../fonts/YesevaOne.otf);
+}
+
+@font-face {
+  font-family: "Lato";
+  src: url(../fonts/Lato.ttf);
+}
+body, html {
+  margin: 0;
+  padding: 0;
+}
+.wrapper {
+  margin: 0 auto;
+  width: 1600px;
+}
+
+.container h1 {
+  font-family: "Yeseva One", sans-serif;
+  display: block;
+  text-align: center;
+  font-size: 90px;
+  color: white;
+  margin: 0;
+  font-weight: 400;
+  text-shadow:  1px 1px 2px black;
+}
+.background-overlay .container h1:nth-child(2) {
+  margin-bottom: 30px;
+}
+
+div.back-1 {
+    background-image: url(../images/background/Bg.png);
+    width: 1600px;
+    height: 750px;
+    position: absolute;
+    top: 0px;
+}
+
+.navbar .container {
+  margin: 0 175px;
+  height: 110px;
+  
+}
+
+.layout-positioner::after {
+  content: "";
+  display: table;
+  clear: both;
+ }
+
+ div.navbar {
+  background: url(../images/background/shadow.png) no-repeat center center;
+  width: 1600px;
+  height: 159px;
```

Comment:
Height usually should be allocated automatically by content and paddings. Here and everywhere.